src/database.rs

```diff
@@ -212,17 +265,22 @@ impl Database {
 
 impl Drop for Database {
     fn drop(&mut self) {
-        unsafe {
-            release(self._ref)
+        if self.has_ownership {
+            unsafe {
+                release(self._ref)
+            }
         }
     }
 }
 
 
-impl Clone for Database {
+/*impl Clone for Database {
```

Comment:
Why you comment `Clone` ?

Reply:
The Drop method was my main issue. Maybe we should set has_ownership to false for all clones to make sure the only owner is the first instance? But in that case the first instance could drop the DB while other instances are unaware.
Maybe a smart pointer could help us out? What do you think?